sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1936,6 +1939,11 @@ class Analyzer(override val catalogManager: CatalogManager) extends RuleExecutor
             case o => o :: Nil
           })
         // count(*) has been replaced by count(1)
```

Comment:
shall we keep this comment above the last default case match?

Reply:
```suggestion
```